boards/arm/mec172xevb_assy6906/mec172xevb_assy6906_defconfig

```diff
@@ -12,6 +12,7 @@ CONFIG_RTOS_TIMER=y
 CONFIG_CLOCK_CONTROL=y
 CONFIG_GPIO=y
 CONFIG_PINMUX=y
+CONFIG_PINCTRL=y
```

Comment:
looks like this change should not be in first commit

Reply:
Only Pin control driver gets compiled on enabling this in the first commit. Ensured all commits are compiling before adding new commits.